data/make_big_raw_image_list.py

```diff
@@ -55,31 +55,38 @@ def load_df(fn, usr):
 print("Values for 'Time-lapse' are:")
 print(time_lapse_values)
 
-print("Might need to update negative values if there are new ones, e.g. 'nope' or 'naw'")
-negative_values = set(['no', 'NO', 'No ', ' no', 'No', 'nan', np.nan, 'No and yes', '  '])
+print(
+    "Might need to update negative values if there are new ones, e.g. 'nope' or 'naw'"
+)
+negative_values = set(
+    ["no", "NO", "No ", " no", "No", "nan", np.nan, "No and yes", "  "]
```

Comment:
isn't there a python func to strip/trim whitespace ?  then you can make your `negative_values` list shorter.  Also convert to upper or lower case.

Reply:
You are correct. I didn't write this file however so feel I shouldn't touch it. It was changed due to `black` formatting. :/

Reply:
Totally could change. The rest of the file is trying to minimally 'bruise' the data as it is loaded so I didn't do any stripping/conversions...

Reply:
for PR readability, code reformatting should be a separate PR from the dask changes if at all possible (for future reference)

Reply:
Totally agree. It was my bad on committing the changes that black made.